executor/slow_query.go

```diff
@@ -490,6 +491,34 @@ func getLineIndex(offset offset, index int) int {
 	return fileLine
 }
 
+// splitByColon split a line like "field: value field: value..."
+func splitByColon(line string) (fields []string, values []string) {
+	re := regexp.MustCompile(`\w+: `)
```

Comment:
Please move this `re` as a global variable, so that it is compiled only once, for example:

https://github.com/pingcap/tidb/blob/master/types/time.go#L3155

Reply:
Good idea!